cms/djangoapps/contentstore/views/course.py

```diff
@@ -283,6 +294,28 @@ def course_rerun_handler(request, course_key_string):
             })
 
 
+@login_required
+@ensure_csrf_cookie
+@require_GET
+def course_index_handler(request, course_key_string):
```

Comment:
I think this should be renamed, as we already have a ``course_index`` handler which generates the course's index page. Maybe ``course_search_index_handler``?